qa/acceptance-tests/src/test/java/io/camunda/it/client/UsageMetricsTest.java

```diff
@@ -72,7 +72,7 @@ private static void waitForUsageMetrics(
       final OffsetDateTime endTime,
       final Consumer<UsageMetricsStatistics> fnRequirements) {
     Awaitility.await("should export metrics to secondary storage")
-        .atMost(EXPORT_INTERVAL.multipliedBy(10))
+        .atMost(EXPORT_INTERVAL.multipliedBy(2))
```

Comment:
👍 Nice, it is better to keep this at minimum. Higher timeout might hide some issues in the future.